src/js/common/utils/validations.js

```diff
@@ -95,7 +95,7 @@ function isValidDateOver17(day, month, year) {
 }
 
 function isValidName(value) {
-  return /^[a-zA-Z][a-zA-Z '\-]*$/.test(value);
```

Comment:
I think the extra `[a-zA-Z]` means the string has to at least have one letter at the start. With the new regex, the name could start with ` '\-` or be empty.